x-pack/plugins/ml/server/routes/data_frame.js

```diff
@@ -35,6 +35,21 @@ export function dataFrameRoutes(server, commonRouteConfig) {
     }
   });
 
+  server.route({
+    method: 'GET',
+    path: '/api/ml/_data_frame/transforms/{jobId}/_stats',
+    handler(request) {
+      const callWithRequest = callWithRequestFactory(server, request);
+      const { jobId } = request.params;
+      console.warn('route', jobId);
```

Comment:
was this `console.warn` left in by mistake?

Reply:
Thanks, yes that was an oversight. Fixed in 498b30b.